packages/rosaenlg-lib/src/LanguageFrench.ts

```diff
@@ -341,9 +345,11 @@ export class LanguageFrench extends LanguageImpl {
       }
 
       if (sentenceParams.negative) {
-        modifiedVerbalGroup.negativeAdverb = sentenceParams.negativeAdverb || 'pas';
+        modifiedVerbalGroup.negativeAdverb = sentenceParams.negativeAdverb ?? 'pas';
       }
 
+      modifiedVerbalGroup.modifierAdverb = sentenceParams.modifierAdverb;

```

Comment:
raise an error when `modifierAdverb` is used in English (in `LanguageEnglish.ts`)